src/Microsoft.AspNetCore.Mvc.DataAnnotations/CompositeValidationAttribute.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Microsoft.AspNetCore.Mvc.DataAnnotations
+{
+    /// <summary>
+    /// Abstract class for grouping attributes of type <see cref="ValidationAttribute"/> into
+    /// one <see cref="Attribute"/>
+    /// </summary>
+    public abstract class CompositeValidationAttribute : Attribute
```

Comment:
Could you rename this to `ValidationProviderAttribute`. We want to steer away from using any name DataAnnotations could add in the future.

Reply:
cc @Eilon \ @mkArtakMSFT  for name